sys-kernel/genkernel/genkernel-9999.ebuild

```diff
@@ -103,7 +103,7 @@ HOMEPAGE="https://wiki.gentoo.org/wiki/Genkernel https://gitweb.gentoo.org/proj/
 LICENSE="GPL-2"
 SLOT="0"
 RESTRICT=""
-IUSE="ibm +firmware"
+IUSE="ibm +firmware +doc -btrfs -plymouth -b2sum -bcache -dmraid -e2fsprogs -iscsi -luks -lvm -mdadm -multipath -ssh -strace -unionfs -xfsprogs -zfs"
```

Comment:
-x is (mostly) the same as x in IUSE, see https://devmanual.gentoo.org/general-concepts/use-flags/index.html#iuse-defaults.